tools/PI/DevHome.PI/Helpers/ExternalTool.cs

```diff
@@ -60,27 +60,26 @@ public string OtherArgs
     [property: JsonIgnore]
     private ImageIcon? _menuIcon;
 
-    [JsonIgnore]
-    private SoftwareBitmap? _softwareBitmap;
-
     public ExternalTool(
         string name,
         string executable,
-        ExternalToolArgType argtype,
-        string argprefix = "",
-        string otherArgs = "",
+        ToolActivationType activationType,
+        string arguments = "",

```

Comment:
Please use string.empty.

Reply:
Default parameter values must be a compile-time constant, string.Empty won't work.